### ISSUE TITLE
Fix menus following recent change to Vector

### ISSUE THREAD

**Issue by MusikAnimal** [azatoth/twinkle#713]
https://gerrit.wikimedia.org/r/c/mediawiki/skins/Vector/+/537769/ removed the styling on `.vectorMenu ul`, so now Twinkle shows bullet points and margins in the dropdowns (temporarily fixed on enwiki with [Special:Diff/918221349](https://en.wikipedia.org/wiki/Special:Diff/918221349)).

`Twinkle.addPortlet()` is a little hard to decipher. Basically what we need to do for Vector is remove the outer div, and put `.menu` on the `ul` element itself.

Markup should change from:
```
<div class="menu">
    <ul>
        ...
    </ul>
</div>
```
to
```
<ul class="menu">
    ...
</ul>
```

The problem is the outer div is apparently used by other skins, and even on Vector for `.vectorTabs` (you can make Twinkle use tabs instead of menus...?).

I will try to fiddle around with a bit more when I have time, but if anyone feels inclined to help it'd be appreciated.

**Comment by Amorymeltzer** [comment]
@MusikAnimal I've only looked at this briefly, but isn't the `div` needing refactoring `innerDiv`, not `outerDiv`?  The former is only added if `type === 'menu'`, which is only for Vector, right?  `blame` seems to bear this out.  What happens if we just remove it and through the `menu` class on the `ul`?

**Comment by MusikAnimal** [comment]
> I've only looked at this briefly, but isn't the `div` needing refactoring `innerDiv`, not `outerDiv`? The former is only added if `type === 'menu'`, which is only for Vector, right? `blame` seems to bear this out. What happens if we just remove it and throw the `menu` class on the `ul`?

I also only looked very briefly :) I did not do any sort of testing, I just wanted to document the issue. I'll try to revisit it soon.

**Comment by siddharthvp** [comment]
I looked into this and have fixed it, before beginning work on #716.

**Comment by Amorymeltzer** [comment]
@MusikAnimal With the change in #763 live, can we remove the temporary fix, or do other menus (moremenu?) still need it?
